src/main/java/io/jenkins/plugins/orka/WaitSSHLauncher.java

```diff
@@ -30,9 +35,14 @@ public WaitSSHLauncher(String host, int sshPort, String vmCredentialsId) {
     public void launch(SlaveComputer slaveComputer, TaskListener listener) throws IOException, InterruptedException {
         int maxRetries = 12;
         int retryWaitTime = 15;
+
         listener.getLogger().println("Waiting for SSH to be enabled");
+        logger.log(Level.INFO, "Waiting for SSH to be enabled");
```

Comment:
Since this will be added to the Jenkins log and not the agent one, we need to be more specific to know "what are we waiting for". That is - we can add host and port to the log.